### PR TITLE
fix: add thalesgomest

### LINKED ISSUE #1563: [PROFILE ADDITION] Thales Gomes Targino
### Profile screenshot

image

### PR DESCRIPTION
**Pull request by thalesgomest**
resolves #1563

### CODE REVIEW

File: profiles/thalesgomest.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "image": "https://user-images.githubusercontent.com/97575616/207838744-93e185a3-7959-46d8-82db-f7d69e843ee8.png",
+  "issueId": 1563,
+  "name": "Thales Gomes Targino",
+  "username": "thalesgomest"
+}
+
```
Comment (Panquesito7) [inline profiles/thalesgomest.json:7]: There's an extra line here.
```suggestion
```

File: profiles/thalesgomest.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "image": "https://user-images.githubusercontent.com/97575616/207838744-93e185a3-7959-46d8-82db-f7d69e843ee8.png",
```
Comment (Panquesito7) [inline profiles/thalesgomest.json:2]: I recommend more of an image like this. What do you think? 🙂

image

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/207966988-9a959e22-a0cc-4393-a54d-04eda0e241fb.png",
```

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**